backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
```

Comment:
are we actually running this docker container? just curious bc it looks like flask run is the command, but we end up using gunicorn

Reply:
No, the Dockerfiles are remnants that were replaced by the app.yaml files. Dockerfiles are removed.